test/expect/TestFuser.test_lstm_cuda-backward.expect

```diff
@@ -25,28 +25,28 @@ graph(%0 : Float(*, *)
       %24 : int[]
       %25 : int[]
       %26 : Float(*, *)) {
-  %27 : int[] = aten::size(%26)
+  %27 : Float(*, *) = aten::mul(%0, %26)
   %28 : int[] = aten::size(%outgate)
-  %29 : int[] = aten::size(%cellgate)
-  %30 : int[] = aten::size(%ingate)
-  %31 : int[] = aten::size(%9)
-  %32 : int[] = aten::size(%forgetgate)
-  %33 : Float(*, *) = aten::mul(%0, %26)
-  %34 : Tensor = prim::SumToSize(%33, %28)
-  %35 : Float(*, *) = aten::mul(%0, %outgate)
-  %36 : Tensor = prim::SumToSize(%35, %27)
-  %37 : Tensor = prim::FusionGroup_0(%1, %36, %26)
-  %38 : Tensor = prim::SumToSize(%37, %24)
-  %39 : Tensor = prim::SumToSize(%37, %25)
-  %40 : Tensor = aten::mul(%39, %cellgate)
-  %41 : Tensor = prim::SumToSize(%40, %30)
-  %42 : Tensor = aten::mul(%39, %ingate)
-  %43 : Tensor = prim::SumToSize(%42, %29)
-  %44 : Tensor = aten::mul(%38, %9)
-  %45 : Tensor = prim::SumToSize(%44, %32)
-  %46 : Tensor = aten::mul(%38, %forgetgate)
-  %47 : Tensor = prim::SumToSize(%46, %31)
-  %48 : Tensor = prim::FusionGroup_1(%41, %ingate, %45, %forgetgate, %43, %cellgate, %34, %outgate)
+  %29 : Tensor = aten::sum_to_size(%27, %28)
+  %30 : Float(*, *) = aten::mul(%0, %outgate)
+  %31 : int[] = aten::size(%26)
+  %32 : Tensor = aten::sum_to_size(%30, %31)
+  %33 : Tensor = prim::FusionGroup_0(%1, %32, %26)
+  %34 : Tensor = prim::SumToSize(%33, %24)
+  %35 : Tensor = prim::SumToSize(%33, %25)
```

Comment:
We should de-dup aten::sum_to_size and prim::SumToSize, but that can be a follow up